mxnet_model_service.py

```diff
@@ -0,0 +1,95 @@
+import mxnet as mx
+import numpy as np
+import tarfile
+import json
+import os
+
+from mxnet.gluon.utils import download
+from mxnet.io import DataBatch
+from model_service import SingleNodeModel, MultiNodesModel
+
+URL_PREFIX = ('http://', 'https://', 's3://')
+MODEL_DIR = 'models/'
```

Comment:
I think after unzipping the model archive for example resnet-18.zip, there will be a resnet-18 folder, I think the model dir will be models/resnet-18.

Reply:
Maybe I should change the name to MODEL_ROOT_DIR, since it is the root folder of model.